src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexInterpreter.cs

```diff
@@ -815,36 +769,35 @@ protected override void Go()
                         continue;
 
                     case RegexCode.Setcount | RegexCode.Back:
-                        StackPop(2);
-                        break;
-
                     case RegexCode.Nullcount | RegexCode.Back:
+                    case RegexCode.Setjump | RegexCode.Back:
                         StackPop(2);
                         break;
 
                     case RegexCode.Branchcount:
                         // StackPush:
                         //  0: Mark
                         //  1: Count
+                        StackPop(2);
```

Comment:
Why put this outside of the brackets?

Reply:
Was just trying to minimize what was indented.  I'm also toying with the idea of consolidating all the local definitions to the top of the method, in which case all of these indentations can go away.